src/assets/images/feedback/face-frown.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const FaceFrown = (props) => (
+  <svg
+    viewBox="0 0 24 24"
+    style={{
+      display: 'inline-block',
+      height: '48px',
+      width: '48px',
+    }}
+  >
+    <defs>
+      <path id="path-1" d="M0,0h2v4h-2Z" />
+    </defs>
+    <path fill="none" d="M0,0h24v24h-24Z" />
+    <g fill={props.color}>
```

Comment:
Cool way to manipulate SVG color :smile:

Reply:
Awesome!